source/particle/interpolator/bilinear_least_squares.cc

```diff
@@ -279,9 +280,8 @@ namespace aspect
       ASPECT_REGISTER_PARTICLE_INTERPOLATOR(BilinearLeastSquares,
                                             "bilinear least squares",
                                             "Interpolates particle properties onto a vector of points using a "
-                                            "bilinear least squares method. "
-                                            "Note that deal.II must be configured with BLAS and LAPACK to "
-                                            "support this operation.")
+                                            "bilinear least squares method. Note that deal.II must be configured "
+                                            "with BLAS/LAPACK.")
```

Comment:
I do not see the value in this change, can you remove the change (revert to the state before)?